app/models/applicant.rb

```diff
@@ -21,4 +22,21 @@ def email_address
   def full_name
     "#{first_name} #{last_name}".strip
   end
+
+  def store_true_layer_token(token:, expires:)
+    data = { token: token, expires: expires }
+    update!(true_layer_secure_data_id: SecureData.create_and_store!(data))
```

Comment:
Do we need to use `SecureData` for that?
At the moment, `BankDataImportService` saves these in clear in `BankProvider`

Reply:
Those tokens allow you to access someones bank account details. I think we should make it difficult for someone to just grab tokens from the database and start trawling the API for bank data.

Reply:
I've modified `BankProvider` so that it stores the id of the applicant's `SecureData`. That will stop people being able to access users data from keys pulled direct from a database dump, and still allow the keys to be retrieved in the future if that is needed.